components/notifications-service/server/lib/formatters/compliance_helper.ex

```diff
@@ -99,4 +102,32 @@ defmodule Notifications.Formatters.ComplianceHelper do
   end
   # Anything that's not a failure gets excluded
   defp prune_results([_| results], acc), do: prune_results(results, acc)
+
+  @spec get_servicenow_compliance_notification(ComplianceFailure.t):: map()
+  def get_servicenow_compliance_notification(notification) do
+    get_servicenow_compliance_notification(notification, @crit_control_threshold)
+  end
+  def get_servicenow_compliance_notification(%ComplianceFailure{test_totals: totals} = notification, control_threshold) do
+    Utils.to_map(notification)
+
+    notification = prune_and_augment(notification, control_threshold)
+    %{
+      automate_fqdn: Notifications.Config.automate_fqdn,
+      failure_snippet: "InSpec found a critical control failure on [#{notification.node_name}](#{notification.compliance_url})",
```

Comment:
`critical` should be removed from this string now because the notification could be noncritical.

Reply:
Done